src/backend/main.ts

```diff
@@ -296,12 +298,21 @@ async function initializeWindow(): Promise<BrowserWindow> {
 
   setTimeout(async () => {
     // Will download Wine if none was found
-    const availableWine = await GlobalConfig.get().getAlternativeWine()
+    const availableWine = (await GlobalConfig.get().getAlternativeWine()) || []
+    const toolkitListDownloaded = availableWine.some(
+      (wine) => wine.type === 'toolkit'
+    )
+    const shouldDownloadWine =
+      !availableWine.length || (isMac && !toolkitListDownloaded)
+
     Promise.all([
       DXVK.getLatest(),
       Winetricks.download(),
-      !availableWine.length ? downloadDefaultWine() : null
+      shouldDownloadWine ? downloadDefaultWine() : null
```

Comment:
Although we check for GPTK here, on the downloadDefaultWine we will skip GPTK and download wine for non-compatible macOS.